Add missing dependency on glusterfs-selinux

sandrobonazzola opened the pull request:
Add missing dependency on glusterfs-selinux.
Fixes: #2780
Bug-Url: https://bugzilla.redhat.com/show_bug.cgi?id=2002178

sandrobonazzola committed:
Add missing dependency on glusterfs-selinux

Add missing dependency on glusterfs-selinux.
Fixes: #2780
Bug-Url: https://bugzilla.redhat.com/show_bug.cgi?id=1761088

gluster-ant commented:
Can one of the admins verify this patch?

gluster-ant commented:
Can one of the admins verify this patch?

gluster-ant commented:
Can one of the admins verify this patch?

sandrobonazzola commented:
@nixpanic can you please review?

mykaul commented:
Looks similar to https://review.gluster.org/#/c/glusterfs/+/24876/ - see comments on that patch.

sandrobonazzola committed:
Address late comments on glusterfs-selinux fix

Address comments arrived on https://review.gluster.org/#/c/glusterfs/+/24876/8/glusterfs.spec.in
after tha patch was merged.

sandrobonazzola commented:
Thanks @mykaul , addressed comments from the referenced patch.

amarts commented:
/run regression
